prow/plugins/project/project_test.go

```diff
@@ -394,6 +394,55 @@ func TestParseCommand(t *testing.T) {
 			proposedColumn:  "",
 			shouldClear:     false,
 		},
+		{
+			hasMatches:      true,
+			command:         "/project '0.0.0'",
+			proposedProject: "0.0.0",
+			proposedColumn:  "",
+			shouldClear:     false,
+		},
+		{
+			hasMatches:      true,
+			command:         "/project \"0.0.0\"",
+			proposedProject: "0.0.0",
+			proposedColumn:  "",
+			shouldClear:     false,
+		},
+		{
+			hasMatches:      true,
+			command:         "/project '0.0.0' To do",
+			proposedProject: "0.0.0",
+			proposedColumn:  "To do",
+			shouldClear:     false,
+		},
+		{
+			hasMatches:      true,
+			command:         "/project '0.0.0' \"To do\"",
+			proposedProject: "0.0.0",
+			proposedColumn:  "To do",
+			shouldClear:     false,
+		},
+		{
+			hasMatches:      true,
+			command:         "/project 'something 0.0.0' \"To do\"",
+			proposedProject: "something 0.0.0",
+			proposedColumn:  "To do",
+			shouldClear:     false,
+		},
+		{
+			hasMatches:      true,
```

Comment:
Can we add a test case for `/project clear project name`?

Reply:
good point, added